.github/workflows/cr.staging.deploy.yaml

```diff
@@ -0,0 +1,74 @@
+name: Deploy to Amazon ECS (STAGING)
```

Comment:
Same comments as in `cr.qa.depliy.yaml`.

Reply:
done